pkg/controller/daemon/daemoncontroller.go

```diff
@@ -199,6 +199,8 @@ func NewDaemonSetsController(podInformer framework.SharedIndexInformer, kubeClie
 			UpdateFunc: dsc.updateNode,
 		},
 	)
+	dsc.nodeStoreSynced = dsc.nodeController.HasSynced
```

Comment:
> Do we really need those variables in dsc?

They're using so that they can mock the startup.  At some point I think we could refactor the test run via a fake client with a fake watch, but its non-trivial to do so.